hack/generate/changelog/gen-changelog.go

```diff
@@ -56,9 +56,6 @@ func main() {
 	if err != nil {
 		log.Fatalf("flag '-tag' is not a valid semantic version: %v", err)
 	}
-	if len(version.Pre) > 0 || len(version.Build) > 0 {
```

Comment:
In the blang/semver library, build information is preceded by `+`, which I think we still don't want in tags.

Reply:
So you're thinking, do this?

```go
if len(version.Build) > 0 {
    log.Fatal( ... )
}
```